cmd/admin-router.go

```diff
@@ -201,6 +201,7 @@ func registerAdminRouter(router *mux.Router, enableConfigOps, enableIAMOps bool)
 
 		// -- KMS APIs --
 		//
+		adminRouter.Methods(http.MethodPost).Path(adminVersion + "/kms/status").HandlerFunc(httpTraceAll(adminAPI.KMSStatusHandler))
```

Comment:
What is the difference between kms/key/status and kms/status? @aead

Reply:
The `kms/status` API returns general information about the KMS - it's endpoint(s), the default KMS key used for SSE-S3 and SSE-KMS when no explicit KMS key id is specified by a user / no SSE-KMS bucket configuration is set, etc.

The `kms/key/status` returns information about a particular key - like whether you can encrypt and decrypt.

Reply:
From the `mc` side think of it as:
```
$ mc kms status <alias>
{
    "DefaultKeyID": "my-minio-key",
    "Endpoints:": {
          "https://kes1.local": "online",
     },
     ...
}

$ mc kms key create <alias> my-key

$ mc kms key status <alias> my-key
my-key:
   - Encrypt: ✓
   - Decrypt: ✓
```

Reply:
Understood @aead is this similar to `mc admin info` is doing `fetchKMSStatus()` ?

Reply:
Yeah - basically KMS-specific and with more user-friendly output.

Reply:
Can your function be re-purposed for `mc admin info` - fetchKMSStatus() ?

Reply:
Sure, that can be done 👍